qt/python/mantidqt/test/test_algorithm_dialog.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+
+from qtpy.QtWidgets import QWidget, QLineEdit
+
+from mantid.api import PythonAlgorithm
+from mantid.kernel import Direction, FloatArrayProperty
+from mantid.simpleapi import AlgorithmManager, AlgorithmFactory
+from mantidqt.utils.qt.plugins import setup_library_paths
+from mantidqt.utils.qt.testing import gui_test_case
```

Comment:
I had to rename `gui_test_case` to remove the `_test_` to be compatible be python's `nose` testing framework in my branch, which I think you have merged in. I've renamed it to `requires_qapp`.